server/sonar-server/src/main/java/org/sonar/server/project/ws/CreateAction.java

```diff
@@ -96,6 +96,14 @@ public void define(WebService.NewController context) {
       .setDescription("SCM Branch of the project. The key of the project will become key:branch, for instance 'SonarQube:branch-5.0'")
       .setExampleValue("branch-5.0");
 
+    action.createParam(PARAM_VISIBILITY)
+      .setDescription("Whether the created project should be visible to everyone, or only specific user/groups.<br/>" +
+        "If no visibility is specified, the default project visibility of the organization will be used.")
+      .setRequired(false)
+      .setInternal(true)
+      .setSince("6.4")
+      .setPossibleValues("private", "public");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "private" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)